<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>foucaultView</class>
 <widget class="QWidget" name="foucaultView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,9">
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>77</width>
        <height>73</height>
       </rect>
      </property>
      <property name="title">
       <string>Source</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="movingSourceRb">
         <property name="text">
          <string>moving</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>Fixed</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>93</x>
        <y>10</y>
        <width>138</width>
        <height>78</height>
       </rect>
      </property>
      <property name="title">
       <string>Foucault</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QCheckBox" name="knifeOnLeftCb">
         <property name="text">
          <string>Knife On Left</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Slit Width</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="slitWidthSb">
           <property name="prefix">
            <string/>
           </property>
           <property name="suffix">
            <string> in</string>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="minimum">
            <double>0.001000000000000</double>
           </property>
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.001000000000000</double>
           </property>
           <property name="value">
            <double>0.006000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>237</x>
        <y>10</y>
        <width>221</width>
        <height>78</height>
       </rect>
      </property>
      <property name="title">
       <string>Grid/Knife Position</string>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>204</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="useMM">
          <property name="text">
           <string>offset in mm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>ROC</string>
          </property>
          <property name="buddy">
           <cstring>rocOffsetSb</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="rocOffsetSb">
          <property name="suffix">
           <string> in</string>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>-100.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.001000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>49</y>
         <width>115</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Lateral</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="lateralKnifeSb">
          <property name="specialValueText">
           <string/>
          </property>
          <property name="suffix">
           <string> in</string>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>-1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.001000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gridGroupBox">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>104</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Ronchi LPI</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QDoubleSpinBox" name="lpiSb">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="maximum">
          <double>300.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>120.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="clearCenterCb">
         <property name="text">
          <string>Clear Center</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="makePb">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>60</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Make</string>
      </property>
     </widget>
     <widget class="QPushButton" name="scanPb">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Scan</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>10</y>
        <width>120</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>Surface Height</string>
      </property>
      <widget class="QRadioButton" name="h1x">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>1X</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="h2x">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>2X</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="h4x">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>4x</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>60</y>
        <width>96</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Gamma</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="gammaSb">
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>2.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <zorder>groupBox_4</zorder>
     <zorder>groupBox_2</zorder>
     <zorder>groupBox_2</zorder>
     <zorder></zorder>
     <zorder>gridGroupBox</zorder>
     <zorder>gridGroupBox</zorder>
     <zorder>groupBox</zorder>
     <zorder>groupBox_4</zorder>
     <zorder>groupBox_2</zorder>
     <zorder>gammaSb</zorder>
     <zorder>label_4</zorder>
     <zorder>gridGroupBox</zorder>
     <zorder>makePb</zorder>
     <zorder>scanPb</zorder>
     <zorder>groupBox_3</zorder>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>66</number>
     </property>
     <item>
      <widget class="QLabel" name="ronchiViewLb">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>4</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Ronchi Display</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="foucaultViewLb">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>4</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Foucaut display</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>ronchiView</zorder>
  <zorder>foucaultView_2</zorder>
  <zorder>makePb</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>gridGroupBox</zorder>
  <zorder>gammaSb</zorder>
  <zorder>label_4</zorder>
  <zorder>useSANull</zorder>
  <zorder>label</zorder>
  <zorder>rocOffsetSb</zorder>
  <zorder>slitWidthSb</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>groupBox_5</zorder>
  <zorder>groupBox</zorder>
  <zorder>foucaultViewLb</zorder>
  <zorder>ronchiViewLb</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
